### PR TITLE
Update LICENSE

### LINKED ISSUE #331: License missing copyright
Hello! Thanks for your work on this library.

Here's an opportunity to update the oldest and least updated file in the repo!

I wanted to properly credit Valet in my app's open source attributions section. When looking though your license file, I saw this:

```
Copyright [yyyy] [name of copyright owner]
```

https://github.com/square/Valet/blob/main/LICENSE#L190

Presumably, it should state something like `Copyright 2015 Square, Inc.`. I'll assume this copyright until someone tells me different, or you update the license.

Cheers!
Brandon

### PR DESCRIPTION
**Pull request by dfed**
Resolves #331

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2014 Square, Inc.
```
Comment (kyleve) [inline LICENSE:190]: I wonder if this should actually be `Block. Inc.` now...

**Review by kyleve — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (dfed) [reply LICENSE:190]: Happy to approve the pull request once legal gets back to you 🙂

Comment (kyleve) [reply LICENSE:190]: 😓

Comment (kyleve) [reply LICENSE:190]: (Let me dig in a bit, you've nerd sniped me...)

Comment (dfed) [reply LICENSE:190]: Appreciate you buddy. Apologies for the landmine.

**Pull request merged**